src/components/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,75 @@
+import React ,{ useState } from 'react';
+import "./Sidebar.css"
+import {  FaArchive , FaTrash , FaBars ,FaStickyNote} from 'react-icons/fa'
+import { MdLabel } from "react-icons/md"
+import { NavLink, useNavigate } from 'react-router-dom';
+
+
+function Sidebar() {
+
+const data = [
+    {
+        title: "My Notes",
+        icon : <FaStickyNote/>,
+        link :"/mynotes"
+    },
+    {
+        title: "Archives",
+        icon :  <FaArchive/>,
+        link :"/archive"
+    },
+    {
+        title: "Labels",
+        icon : <MdLabel/>,
+        link :"/label"
+    },
+    {
+        title: "Trash",
+        icon : <FaTrash/>,
+        link :"/trash"
+    },
+]
+
+const [ showSidebar , setShowSidebar ] = useState(false);
+
+    const hamburgHandler = (e)=>{
+        e.preventDefault();
+        if(showSidebar===true){
+        setShowSidebar(false) ;
+        }else{
+            setShowSidebar(true) ;
+        }
+        
+    }
+    console.log(showSidebar);
```

Comment:
remove console logs before pushing the code

Reply:
thanks for pointing out . done